src/System.Runtime/tests/System/ExceptionTests.cs

```diff
@@ -57,6 +59,103 @@ public static void Exception_TargetSite_Aot()
 
             Assert.True(caught);
         }
+
+        [Fact]
```

Comment:
You will need to add `[SkipOnTargetFramework ...` for .NET Framework, until we port this. Usually we note the issue that fixed it for core.

Reply:
Or possibly use PlatformDetection.IsFullFramework to verify .NET Framework behavior, so when it is ported, this test automatically fails and reminds us to enable it